packages/profile-sync-controller/package.json

```diff
@@ -130,7 +130,7 @@
   },
   "peerDependencies": {
     "@metamask/accounts-controller": "^18.1.1",
-    "@metamask/keyring-controller": "^17.2.0",
+    "@metamask/keyring-controller": "^18.0.0",
```

Comment:
Similar, can we include this in the release?

Reply:
Major.

Reply:
Or maybe this one should minor since it's still `0.x`?

Reply:
Yes, but major works too! It's used in production AFAIK, so I don't see why we wouldn't use `1.x`.